### ISSUE TITLE
AssertionError on isinstance(gene_set_db, GeneSetDB)

### ISSUE THREAD

**Issue by alephreish** [flo-compbio/gopca#6]
I'm having trouble running the program: it fails with the following exception even on test datasets.

```
$ uname -a
Linux myserver 3.19.0-47-generic #53~14.04.1-Ubuntu SMP Mon Jan 18 16:09:14 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

$ awk -F'\t' '{print$5}' GO_gene_sets_fly_ensembl83_goa54_ontology2016-01-18.tsv | tr , '\n' | sort | uniq | awk -v OFS='\t' 'BEGIN{print"","A","B","C","D","E"}{printf$1;for(i=1;i<6;i++)printf"\t%d",rand()*100;printf"\n"}' > expression_test.ts
$ head -n5 expression_test.tsv 
    A   B   C   D   E
128up   23  29  84  15  58
14-3-3epsilon   19  81  17  48  15
14-3-3zeta  36  49  91  26  89
18w 22  63  57  33  10

$ go-pca.py -e expression_test.tsv -o test -s GO_gene_sets_fly_ensembl83_goa54_ontology2016-01-18.tsv
[2016-08-04 22:09:49] INFO: Expression matrix size: (p = 6225 genes) x (n = 5 samples).
Traceback (most recent call last):
  File "/usr/local/bin/go-pca.py", line 9, in <module>
    load_entry_point('gopca==1.2.3', 'console_scripts', 'go-pca.py')()
  File "/usr/local/lib/python2.7/dist-packages/gopca/main.py", line 325, in main
    M = GOPCA(config, gene_sets, gene_ontology)
  File "/usr/local/lib/python2.7/dist-packages/gopca/go_pca.py", line 85, in __init__
    assert isinstance(gene_set_db, GeneSetDB)
AssertionError
```

**Comment by flo-compbio** [comment]
Confirmed, I've pushed a fix (version 1.2.4 => install with `pip install --upgrade gopca`).

I'm sorry about really annoying bugs like this -- I'm still actively developing the package (adding features etc.), and I don't have a full suite of tests yet that make sure everything is still working after I change stuff. However, I'm working on adding more tests, so things will get better. In the meantime, I appreciate any bug report and will take care of them immediately.
